netmanager-app/components/sidebar.tsx

```
@@ -0,0 +1,407 @@
+  BarChart2,
+  Users,
+  Shield,
+  Radio,
+  MapPin,
+  Layers,
+  Grid,
+  Building2,
+  Activity,
+  UserCircle,
+  Download,
+  Map,
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)